Stop eagerly after N solvers being scheduled

fridex opened the pull request.

fridex committed:
Stop eagerly after N solvers being scheduled

ghost commented:
Build failed.

- [thoth-coala ](https://zuul.thoth-station.ninja/logs/05/105/a6385118eb60c8d475283ecdfc79ad9ed04f5a45/check/thoth-coala/dfe0461/report.html) : FAILURE in 3m 26s

goern committed:
Update app.py

fixing
|  60| def·graph_refresh(graph_hosts:·str·=·None,·graph_port:·int·=·None)·->·None:
|    | [NORMAL] PycodestyleBear (E302):
|    | E302 expected 2 blank lines, found 1

ghost commented:
Build succeeded.

- [thoth-coala ](https://zuul.thoth-station.ninja/logs/05/105/fa4c543514b7b6612bbd6558fd4cba3338931ba6/check/thoth-coala/0e5844f/) : SUCCESS in 1m 19s

ghost commented:
Build succeeded (gate pipeline).

- [thoth-coala ](https://zuul.thoth-station.ninja/logs/05/105/fa4c543514b7b6612bbd6558fd4cba3338931ba6/gate/thoth-coala/b4bffd0/) : SUCCESS in 2m 31s
